### PR TITLE
Tabs - watch the active prop

### PR DESCRIPTION
**Pull request by antennaio**
Watch the `active` prop and change the active tab accordingly.

### CODE REVIEW

**Comment by antennaio** [comment]
[(2) Tabs - watch the active prop](https://trello.com/c/sm3pGjCA/3003-2-tabs-watch-the-active-prop)

File: dist/vue-bootstrap.js [hunk 1]
```diff
@@ -1 +1 @@
...
...encodeURIComponent(JSON.stringify(o))))+" */");if(t.styleSheet)t.styleSheet.cssText=n;else{for(;t.firstChild;)t.removeChild(t.firstChild);t.appendChild(document.createTextNode(n))}}.bind(null,i),n=function(){i.parentNode.removeChild(i)};return e(t),function(i){if(i){if(i.css===t.css&&i.media===t.media&&i.sourceMap===t.sourceMap)return;e(t=i)}else n()}}var g,b=(g=[],function(t,e){return g[t]=e,g.filter(Boolean).join("\n")});function _(t,e,n,i){var o=n?"":i.css;if(t.styleSheet)t.styleSheet.cssText=b(e,o);else{var a=document.createTextNode(o),r=t.childNodes;r[e]&&t.removeChild(r[e]),r.length?t.insertBefore(a,r[e]):t.appendChild(a)}}},function(t,e,n){"use strict";n.d(e,"a",function(){return a});var i="v-bs-click-outside-handler",o=function(t){document.documentElement.removeEventListener("click",t[i],!1),delete t[i]},a={directives:{onClickOutside:{bind:function(t,e){var n=e.value,a=!1;setTimeout(function(){a=!0},0),o(t),t[i]=function(e){if(a&&t!==e.target&&!t.contains(e.target))return n(e)},document.documentElement.addEventListener("click",t[i],!1)},unbind:o}}}},function(t,e,n){"use strict";function i(){var t,e=document.createElement("div");return e.className="modal-scrollbar-measure",d
```
Comment (think-01) [inline dist/vue-bootstrap.js]: should this be commited?

File: docs/js/docs.bundle.js [hunk 2]
```diff
@@ -58,7 +58,7 @@
...
...r: function validator(type) {\n        return ['tabs', 'pills'].includes(type);\n      }\n    },\n    stacked: {\n      type: Boolean,\n      \"default\": false\n    },\n    preselectFirstTab: {\n      type: Boolean,\n      \"default\": true\n    }\n  },\n  watch: {\n    active: function active(id) {\n      this.select(id);\n    }\n  },\n  methods: {\n    findTab: function findTab(id) {\n      return this.tabs.find(function (tab) {\n        return tab.id === id;\n      });\n    },\n    select: function select(id) {\n      var tab = this.findTab(id);\n\n      if (!tab || tab.disabled || tab.id === this.selected) {\n        return;\n      }\n\n      this.selected = tab.id;\n      this.$emit('selected', this.selected);\n    }\n  },\n  created: function created() {\n    if (this.active) {\n      this.select(this.active);\n    } else if (this.preselectFirstTab && this.tabs.length) {\n      this.select(this.tabs[0].id);\n    }\n  }\n});\n\n//# sourceURL=webpack:///./src/Tabs.vue?./node_modules/babel-loader/lib!./node_modules/vue-loader/lib??vue-loader-options")},"./node_modules/babel-loader/lib/index.js!./node_modules/vue-loader/lib/index.js?!./src/TagInput.vue?vue&type=script&lang=js&":
```
Comment (think-01) [inline docs/js/docs.bundle.js]: should this be commited?

File: dist/vue-bootstrap.js (hunk 1 above)
Comment (antennaio) [reply dist/vue-bootstrap.js]: This is the result of running `npm run build`... at some point it needs to be committed, either now or very shortly when a new version is released.

File: docs/js/docs.bundle.js (hunk 2 above)
Comment (antennaio) [reply docs/js/docs.bundle.js]: This is the result of running `npm run docs:build`...

File: dist/vue-bootstrap.js (hunk 1 above)
Comment (think-01) [reply dist/vue-bootstrap.js]: allright just wanted to make sure its intentional - good to go then :+1:

**Pull request merged**